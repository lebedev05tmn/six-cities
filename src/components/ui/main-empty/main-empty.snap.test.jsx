import React from "react";
import {render} from "@testing-library/react";
import MainEmpty from "./main-empty";

test(`Should Main Empty renders correctly`, () => {
  const {container} = render(<MainEmpty />);
  expect(container).toMatchSnapshot();
});
