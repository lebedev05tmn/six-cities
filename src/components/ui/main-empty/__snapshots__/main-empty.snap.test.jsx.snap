// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should Loading Screen renders correctly 1`] = `
<div>
  <div
    class="cities"
  >
    <div
      class="cities__places-container cities__places-container--empty container"
    >
      <section
        class="cities__no-places"
      >
        <div
          class="cities__status-wrapper tabs__content"
        >
          <b
            class="cities__status"
          >
            No places to stay available
          </b>
          <p
            class="cities__status-description"
          >
            We could not find any property available at the moment in Dusseldorf
          </p>
        </div>
      </section>
      <div
        class="cities__right-section"
      />
    </div>
  </div>
</div>
`;
